src/pygama/flow/data_loader.py

```diff
@@ -0,0 +1,1092 @@
+            else:
+                tables = self.fileDB.df.loc[file, f"{self.tiers[low_level][0]}_tables"]
+
+            for tb in tables:
+                tb_table = None
+                if not cut:
+                    tier = self.tiers[low_level][0]
+                    tier_path = (
+                        self.data_dir
+                        + self.fileDB.tier_dirs[tier]
+                        + "/"
+                        + self.fileDB.df.iloc[file][f"{tier}_file"]
+                    )
+                    if os.path.exists(tier_path):
+                        table_name = self.get_table_name(tier, tb)
+                        n_rows = sto.read_n_rows(table_name, tier_path)
+                        tb_df = pd.DataFrame(
+                            {
+                                f"{low_level}_idx": np.arange(n_rows),
+                                f"{low_level}_table": tb,
+                            }
+                        )
+                else:
+                    for tier in self.tiers[low_level]:
+                        if tier in col_tiers[file]["tables"].keys():
+                            tier_path = (
+                                self.data_dir
+                                + self.fileDB.tier_dirs[tier]
+                                + "/"
+                                + self.fileDB.df.iloc[file][f"{tier}_file"]
+                            )
+                            if tier in col_tiers[file]["tables"].keys():
+                                if tb in col_tiers[file]["tables"][tier]:
+                                    table_name = self.get_table_name(tier, tb)
+                                    tier_tb, _ = sto.read_object(
+                                        table_name, tier_path, field_mask=cut_cols
+                                    )
+                                    if tb_table is None:
+                                        tb_table = tier_tb
+                                    else:
+                                        tb_table.join(tier_tb)
+                    tb_df = tb_table.get_dataframe()
+                    tb_df.query(cut, inplace=True)
+                    tb_df[f"{low_level}_table"] = tb
+                    tb_df[f"{low_level}_idx"] = tb_df.index
+                f_entries = pd.concat((f_entries, tb_df), ignore_index=True)[entry_cols]
+            if in_mem:
+                entries[file] = f_entries
+            if f_output:
+                # Convert f_entries DataFrame to Struct
+                f_dict = f_entries.to_dict("list")
+                f_struct = Struct(f_dict)
+                sto.write_object(f_struct, f"entries/{file}", f_output, wo_mode="o")
+        if in_mem:
+            return entries
+
+    def load(
+        self,
+        entry_list: pd.DataFrame = None,
+        in_mem: bool = False,
+        f_output: str = None,
+        rows: str = "hit",
+        tcm_level: str = None,
```

Comment:
Maybe more self-documenting?
```suggestion
        in_memory: bool = False,
        output_file: str = None,
        orientation: str = "hit",
        tcm_level: str = None,
```